src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
So this works for Gentoo, but breaks if I want to have my own ZNC installation in `~/znc-install`? That sounds bad.